src/analyzer/protocol/login/Login.cc

```diff
@@ -564,6 +564,7 @@ void Login_Analyzer::AddUserText(const char* line)
 		if ( ++user_text_last == MAX_USER_TEXT )
 			user_text_last = 0;
 
+		delete[] user_text[user_text_last];
```

Comment:
We came to a conclusion a bit ago that this wasn't right: https://github.com/zeek/zeek/pull/756#discussion_r372101289

Maybe time to leave a comment so we don't accidentally revisit again later?

Reply:
Oops, this was still on my stgit stack, and when I assembled this PR yesterday, it looked so correct... sorry.